include/utils/error/error_database_map.hpp

```diff
@@ -14,6 +14,7 @@ namespace error {
 class ErrorDatabaseMap : public ErrorDatabase {
 public:
     ErrorDatabaseMap() = default;
+    virtual ~ErrorDatabaseMap() = default;
```

Comment:
Is this related to this PR?

Reply:
no - it generates lots of warnings that hide real issues.

Reply:
I'll remove this - now addressed in separate PR #216